docs/reference/how-to/disk-usage.asciidoc

```diff
@@ -0,0 +1,113 @@
+[[tune-for-disk-usage]]
+== Tune for disk usage
+
+=== Disable the features you do not need
+
+By default elasticsearch indexes and adds doc values to most fields so that they
+can be searched and aggregated out of the box. For instance if you have a numeric
+field called `foo` that you need to run histograms on but that you never need to
+filter on, you can safely disable indexing on this field in your
+<<mappings,mappings>>:
+
+[source,js]
+--------------------------------------------------
+{
+  "foo": {
+    "type": "integer",
+    "index": false
+  }
+}
+--------------------------------------------------
```

Comment:
I think it'd be nice to have `// CONSOLE` and full syntax. That way the reader knows which API they are working with. And the snippets are executed during testing to make sure we don't break them.